server/index.ts

```diff
@@ -54,24 +52,29 @@ try {
 } catch (e) {
 	throw new Error("We require Node.js version 10 or later; you're using " + process.version);
 }
+
 try {
 	require.resolve('../.sim-dist/index');
 } catch (e) {
 	throw new Error("Dependencies are unmet; run `node build` before launching Pokemon Showdown again.");
 }
 
-const FS = require('../.lib-dist/fs').FS;
+import {FS} from '../lib/fs';
 
 /*********************************************************
  * Load configuration
  *********************************************************/
 
-const ConfigLoader = require('../.server-dist/config-loader');
+// global becomes much easier to use if declared as an object
+declare var global: any;
```

Comment:
```suggestion
declare const global: any;
```

We shouldn't be changing where it points to, anyway.